Src/CSharpier.Tests/SyntaxNodeComparerTests.cs

```diff
@@ -156,12 +158,14 @@ public DropdownAttribute(bool ignoreIfNotPresent)
         [Test]
         public void MissingSemiColon()
         {
-            var left = @"public enum Enum
+            var left =
+                @"public enum Enum
```

Comment:
Okay, the more I think about it, the more I realize: _there is no good way_ to format multiline strings! :stuck_out_tongue:

Reply:
The recent PR I might actually make it easier to keep the `@"public enum Enum` on the same line, although it may depend on implementing ConditionalGroup